.gitignore

```diff
@@ -17,4 +17,5 @@ kamel
 .kcp/
 .kube/
 .kind/
-.mvn/mvnd.properties
\ No newline at end of file
+.mvn/mvnd.properties
+.ocm.*
```

Comment:
What is this about?

Reply:
If you install the `ocm` command-line tool, then run the `ocm login` with the provided token; a `.ocm.json` file is generated, which I have excluded here.

Reply:
Thx for the clarification. Let's make this explicit then and set `.ocm.json` here.